static/app/views/settings/organizationMembers/organizationMemberDetail.tsx

```diff
@@ -255,175 +244,178 @@ class OrganizationMemberDetail extends DeprecatedAsyncView<Props, State> {
     return false;
   }
 
-  renderMemberStatus(member: Member) {
-    if (this.memberDeactivated) {
+  if (isPending) {
+    return <LoadingIndicator />;
+  }
+
+  if (isError) {
+    return <LoadingError onRetry={refetch} />;
+  }
+
+  if (!member) {
+    return <NotFound />;
+  }
+
+  function renderMemberStatus() {
+    if (memberDeactivated) {
       return (
         <em>
           <DisabledMemberTooltip>{t('Deactivated')}</DisabledMemberTooltip>
         </em>
       );
     }
-    if (member.expired) {
+    if (member!.expired) {
```

Comment:
do we need `!` here? in the line 255 we return `<NotFound />` if it is not available